src/app/pages/swap/swap-choose-asset/components/swap-asset-list.tsx

```diff
@@ -62,16 +63,25 @@ export function SwapAssetList({ assets }: SwapAssetList) {
       setFieldError('swapAmountTo', undefined);
     }
   }
-
+  console.log('selectableAssets', selectableAssets.length - 1);
   return (
-    <SwapAssetListLayout>
-      {selectableAssets.map(asset => (
-        <SwapAssetItem
-          asset={asset}
-          key={asset.balance.symbol}
-          onClick={() => onChooseAsset(asset)}
-        />
-      ))}
-    </SwapAssetListLayout>
+    <Virtuoso
```

Comment:
@fbwoolf , I'm working on this now. As it's a list in a dialog I said I would add `Virtuoso` so it matches up with the others. 

Is that OK? 

It's still a bit buggy but in the old system the header was scrolling out of view anyway:

NEW:

https://github.com/leather-wallet/extension/assets/2938440/6dc0d1c2-1f24-4e21-aaff-bc984ed3b176



OLD: 

https://github.com/leather-wallet/extension/assets/2938440/02c2aeaa-99e1-46ba-8e27-ead4be9a9f6a

Reply:
Yep, seems like they should match up, thx.

Reply:
I'll remove the use of `Virtuoso` here. I added it to make the behaviour of these types of dialogs more consistent but you're right, it's not really needed. 

 Currently in production the header scrolls out of view so I will fix that on it's own and remove Virtusoso

Reply:
I've reverted this to not use Virtuoso in Swap